doc/Language/modules.pod6

```diff
@@ -524,6 +524,106 @@ The directories in the C<RAKULIB> path will be searched for modules when
 Raku C<need>s, C<use>s or C<require>s them. Directories that start with a
 dot are ignored and symlinks are followed.
 
+=head1 Testing modules and a distribution
+
+It is important in this section to repeat the note at the beginning of this document, namely that
+there is a difference between a B<Raku> C<distribution>, which approximates a I<module> in other languages, 
+and a B<Raku> 
+L<module declaration|https://docs.raku.org/language/syntax#index-entry-declarator_unit-declarator_module-declarator_package-Package,_Module,_Class,_Role,_and_Grammar_declaration>.
+The reason for this is that a single file may contain a number of C<class>, C<module>, C<role> etc declarations, or
+these declarations may be spread between different files. In addition, when a C<distribution> is published (see 
+L<Distributing modules>) it may be necessary to provide access to other resources, such as callable utilities.
+
+B<Raku> also allows for a single distribution to provide multiple C<compunit>s that can be I<use'd> (or I<require'd> etc).
+The information
+about a distribution is contained in the C<META6.json> file in the distribution's root directory. See below
+for more about C<META6.json>. Each entity that the B<distribution> allows to be I<use'd> (or I<require'd> etc), 
+also called a C<compunit>, is 
+placed in the C<provides> section of the C<META6.json> (as specified below). 
+
+It should also be noted that when 
+writing the C<depends> section of the C<META6.json> file, it is the name of the B<distribution> that is listed,
+not the name of the C<compunit> that is desired. Although, for very many entities, the name of the I<distribution>
+and the name of the I<compunit> are the same.
+
+Given this arrangement, the B<rakudo> compiler, or package manager, such as B<zef>, can obtain all the information
```

Comment:
Given that now we have some other package managers, maybe say "package manager*s*" and drop the comma.

Reply:
made reference to zef as an example package manager